packages/flame/lib/src/components/text_box_component.dart

```diff
@@ -182,14 +182,23 @@ class TextBoxComponent<T extends TextRenderer> extends PositionComponent {
       return;
     }
     super.render(c);
+    final devicePixelRatio = window.devicePixelRatio;
```

Comment:
can this change? technically we should save the previous pixelRatio and compare to force a redrawing. I am concerned we might get a bug where text suddenly is scaled up or down for no apparent reason

Reply:
https://api.flutter.dev/flutter/dart-ui/FlutterView/devicePixelRatio.html

Reply:
might not be worth the effort then. we can always revisit in case of bugs. is that the same for web? my concern was like resizing a browser or something